javascript/javascript3/week1/weather-app/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>WeatherApp</title>
+  </head>
+  <body>
+    <header>
+      <img src="images/logo.png" alt="logo" />
+    </header>
+    <main>
+      <div class="inputs">
+        <input type="text" />
```

Comment:
If you wrap your fields with a `form` you can use a [`submit`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit_event) event listener on the form element to enable keyboard-submission of the search field and improve accessibility 🤓

Reply:
Thanks for the hint 🤓